sample/src/main/java/com/chuckerteam/chucker/sample/MainActivity.kt

```diff
@@ -97,6 +99,19 @@ class MainActivity : AppCompatActivity() {
             cache("Mon").enqueue(cb)
             cache(30).enqueue(cb)
         }
+
+        with(SampleApiService.getGraphQLInstance(getClient(this))) {
```

Comment:
The method `getClient` doesn't only get the HTTP client, it also create it. It should be named `buildHttpClient` or something, it would explicitly show that something is built by this method. And maybe we shouldn't abuse of calls to this method.